io.openems.edge.io.iot2000/src/io/openems/edge/io/iot2000/api/IOT2000Util.java

```diff
@@ -0,0 +1,253 @@
+package io.openems.edge.io.iot2000.api;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+
+public class IOT2000Util {
+
+	private static final String[] ALLPINS = { "7", "46", "30", "31", "15", "42", "43", "5", "44", "72", "24", "25",
```

Comment:
some numbers or strings that are used more than one time could be replaced by a constat to avoid magic numbers